tests/integration/DOM-test.js

```diff
@@ -184,3 +185,19 @@ test('Clicking modal overlay', function(assert) {
   });
 
 });
+
+test('Pressing escape', function(assert) {
+
+  visit('/');
+
+  showModal(templateName);
+
+  andThen(function() {
+    keyEvent(inspect('overlay'), 'keydown', escapeKeyCode);
```

Comment:
hmm, that's very odd because `showModal` is an async helper too. If the test is working that's good enough for now.